Source/FolioReaderContainer.swift

```diff
@@ -95,8 +95,13 @@ open class FolioReaderContainer: UIViewController {
         
         // If user can change scroll direction use the last saved
         if readerConfig.canChangeScrollDirection {
-            let direction = FolioReaderScrollDirection(rawValue: FolioReader.currentScrollDirection) ?? .vertical
-            readerConfig.scrollDirection = direction
+            var scrollDirection = FolioReaderScrollDirection(rawValue: FolioReader.currentScrollDirection) ?? .vertical
```

Comment:
This loads the persisted scroll direction, which, in the first app run, is `.defaultVertical` as set by the line #72.